examples/net_framework/DeepSpeechWPF/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CSCore" version="1.2.1.2" targetFramework="net462" />
-  <package id="NAudio" version="1.8.5" targetFramework="net462" />
+  <package id="DeepSpeech" version="0.5.0-alpha.10" targetFramework="net462" />
```

Comment:
We can't have the version here

Reply:
As this is the example we use the DS client nugget with the version that is working for that example. I think is better to avoid the local client since we are updating it without testing the example for the new version.